muikku-core-plugins/src/main/resources/fi/otavanopisto/muikku/plugins/profile/ProfileJsPluginMessages_fi.properties

```diff
@@ -30,4 +30,8 @@ plugin.profile.changeAddressMunicipality.dialog.municipalityField.label = Kotiku
 plugin.profile.changeImage.dialog.title = K�ytt�j�profiilin kuva
 plugin.profile.changeImage.dialog.saveButton.label = Tallenna
 plugin.profile.changeImage.dialog.cancelButton.label = Peruuta
-plugin.profile.properties.saved = Tiedot tallennettu
\ No newline at end of file
+plugin.profile.properties.saved = Tiedot tallennettu
+plugin.profile.chat.visibility = Chat-n�kyvyys
+plugin.profile.chat.visibleToAll = Chat k\u00E4yt\u00F6ss\u00E4
```

Comment:
Weird encoding for Scandinavian characters. Locale files should be UTF8 so using plain Scandinavian characters should work just fine.